README.md

```diff
@@ -62,6 +62,45 @@ The following table shows which APIs are linked to which forms. To download the
 Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+### 9 April 2024
+
+The following 8 new APIs have been created and are now available in Sandbox as part of the breakup of 
+`individuals-income-received-api`:
+
+#### individuals-employments-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to
+- `employment income`
+- `other employment income`
+- `Non-PAYE employment income`
+
+#### individuals-dividends-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to `dividends income`.
+
+#### individuals-foreign-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to `foreign income`.
+
```

Comment:
Why are 'foreign income', 'insurance policies income' etc formatted as code with backticks? No need to do that

Reply:
Removed the backticks